README-CN.md

```diff
@@ -364,6 +364,19 @@ ROS 模板的示例和最佳实践。模板分类如下：
 <details>
   <summary>documents</summary>
 
+- alb
+
+| 模板                                                                                                                 | 说明                 |
+|--------------------------------------------------------------------------------------------------------------------|--------------------|
+| [alb-create-ipv4-load-balancer-and-dnsrecord.yml](./documents/alb/alb-create-ipv4-load-balancer-and-dnsrecord.yml) | 快速实现IPv4服务的负载均衡。 \ | [教程](https://help.aliyun.com/document_detail/210822.html)|
```

Comment:
Please use the international site document URL. For example: [https://www.alibabacloud.com/help/server-load-balancer/latest/application-load-balancer-quick-start](https://www.alibabacloud.com/help/server-load-balancer/latest/application-load-balancer-quick-start)